src/api/routes/authRouter.js

```diff
@@ -1,10 +1,12 @@
 import { Router } from 'express';
 import ValidateUser from '../../middlewares/ValidateUser';
 import authController from '../controllers/authController';
+import EmailVerifier from '../controllers/verifyEmail';
 
 const authRouter = new Router();
 
 authRouter.post('/signup', ValidateUser.validateSignup, authController.signup);
 authRouter.post('/login', authController.login);
+authRouter.get('/verifyEmail/:token', EmailVerifier.verifyEmail);
```

Comment:
@truestbyheart in line 42 of src/api/controllers/authController.js above, the url is `${env.baseUrl}/users/verifyEmail` but here it is `/verifyEmail/:token`, kindly look into this.

Reply:
@truestbyheart kindly respond to my previous comments so I know you've seen them.